certified-connectors/ZohoSign/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1671 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Noc-2021",
```

Comment:
Hi @ZohoSign-integ, you are completely changing the name of the connector, which is not allowed. can you please explain why?. If this is a new connector then please register it by filling our form. Otherwise please use the old title. apologies for the inconvenience this may cause.